### PR TITLE
pass thru feature `nix-experimental`

### PR DESCRIPTION
**Pull request by KiaraGrouwstra**
I found that the [`nickel` package in nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/ni/nickel/package.nix) had yet to add support for nickel's [`nix-experimental`](https://github.com/tweag/nickel/commit/91d63ed611f2f86e245b696b34b4cf09f637ee91) feature flag.

in an [in-progress attempt](https://github.com/NixOS/nixpkgs/compare/master...KiaraGrouwstra:nixpkgs:nickel-enable-nix-import?expand=1) to add this, i bumped into a few hurdles for this on the nickel side.
this PR tries to address these, on the Cargo side erring toward passing thru the feature for a few extra packages for consistency.

whereas i'm not sure this would be sufficient on the nickel side (yet having to get it to work at run-time), i felt maybe i could get a bit of feedback on this direction already.

### CODE REVIEW

File: wasm-repl/Cargo.toml [hunk 1]
```diff
@@ -14,4 +14,7 @@ nickel-lang-core = { workspace = true, default-features = false, features = ["re
 
 [lib]
 crate-type = ["cdylib", "rlib"]
-bench = false
\ No newline at end of file
+bench = false
+
+[features]
+nix-experimental = ["nickel-lang-core/nix-experimental"]
```
Comment (jneem) [inline wasm-repl/Cargo.toml:20]: I understand the other ones, but will nix really build for wasm?

Comment (KiaraGrouwstra) [reply wasm-repl/Cargo.toml:20]: [this seems to build, so i guess so?](https://github.com/NixOS/nixpkgs/compare/master...KiaraGrouwstra:nixpkgs:nickel-enable-nix-import-wasm?expand=1) 🤷‍♀️🤷‍♀️

Comment (jneem) [reply wasm-repl/Cargo.toml:20]: Fair enough. It does fail, though, if I add `--target wasm32-unknown-unknown` and use that together with the "nix-experimental" feature.

Comment (KiaraGrouwstra) [reply wasm-repl/Cargo.toml]: fair! i'll just remove it then

**Review by jneem — APPROVED** [review]

**Comment by KiaraGrouwstra** [comment]
corresponding [nixpkgs PR](https://github.com/NixOS/nixpkgs/pull/384215)

**Pull request merged**